packages/tabs/index.tsx

```diff
@@ -25,20 +27,26 @@ interface TabsProps {
   centered?: boolean
   /** Optional className to add to the root element. */
   className?: string
+  /** Optional className to add to the tabs bar container. Useful for sticky positioning. */
+  classNameTabBar?: string
```

Comment:
This new prop allows consumers to pass a `className` to the tab bar container. Doing this out of necessity - we already have multiple use cases where consumers have had to override the same element in order to implement "sticky" tabs.